scrapers/MatureNL.yml

```diff
@@ -0,0 +1,43 @@
+name: "Mature.nl"
+sceneByURL: &byUrl
+  - action: scrapeXPath
+    url:
+      - mature.nl/
+    scraper: sceneScraper
+galleryByURL: *byUrl
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &title //div[@class="box"]/h1/text()
+      Details: &details //meta[@name="description"]/@content
```

Comment:
It seems that some scenes (only found one atm) have an actual description that should be prefered over the meta
sample `https://www.mature.nl/en/update/14026/spying-on-my-older-and-taller-stepsister`
Based on the above the following selector seems to work ok. I am not sure if the `synopsis` part is there in other scenes though. We would have to find some more with actual descriptions
```yaml
      Details: &details //div/span[text()="Synopsis:"]/following-sibling::text()|//meta[@name="description"]/@content
```

Reply:
Yeah found scenes with the Synopsis part with [Google](https://www.google.com/search?q=site%3Amature.nl+%22synopsis%3A%22) quite easily. 

Your suggestion seems to be good and works with both kinds of pages. I'll add the change.